shared/src/business/entities/trialSessions/IrsCalendarAdministratorInfo.ts

```diff
@@ -0,0 +1,32 @@
+import { ExcludeMethods } from 'types/TEntity';
+import { JoiValidationConstants } from '../JoiValidationConstants';
+import { JoiValidationEntity } from '@shared/business/entities/JoiValidationEntity';
+
+export class IrsCalendarAdministratorInfo extends JoiValidationEntity {
+  public name: string;
+  public email: string;
+  public phone: string;
+
+  constructor(rawData: any) {
+    super('IrsCalendarAdministratorInfo');
+
+    this.name = rawData.name;
+    this.email = rawData.email;
+    this.phone = rawData.phone;
+  }
+
+  static VALIDATIONS = {
+    email: JoiValidationConstants.STRING.max(100).optional(),
```

Comment:
Does it make sense to use `JoiValidationConstants.EMAIL.` here?

Reply:
The AC said they can modify the auto populated email input which use to be just a text area for all 3 inputs I wanted to make sure it continued to work as it did before. The legacy property in TrialSession.ts had only that string max 100 validation.

<img width="572" alt="image" src="https://github.com/ustaxcourt/ef-cms/assets/94860350/000024c4-b238-4b6f-b42d-6bdd2c19cba6">